### PR TITLE
release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
...
 These should not affect the functionality of the module.
 
-## [v1.0.1](https://github.com/voxpupuli/puppet-tea/tree/v1.0.1) (2018-10-13)
+## [v2.0.0](https://github.com/voxpupuli/puppet-tea/tree/v2.0.0) (2020-11-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-tea/compare/v1.0.1...v2.0.0)
+
+**Breaking changes:**
+
+- modulesync 2.7.0 and drop puppet 4 [\#49](https://github.com/voxpupuli/puppet-tea/pull/49) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (igalic) [inline CHANGELOG.md:13]: why is this a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- modulesync 3.0.0 & puppet-lint updates [\#56](https://github.com/voxpupuli/puppet-tea/pull/56) ([bastelfreak](https://github.com/bastelfreak))
+- remove EmailAddress type [\#54](https://github.com/voxpupuli/puppet-tea/pull/54) ([b4ldr](https://github.com/b4ldr))
```
Comment (igalic) [inline CHANGELOG.md:18]: but this isn't?

**Review by igalic — COMMENTED** [review]
🙋🏻‍♀️

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md:13]: we considered dropping puppet 4 a breaking change in all of our modules

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md:18]: oh that pr was just for the docs. I will update the pr title and regenerate the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (igalic) [reply CHANGELOG.md:13]: that deserves its own entry, not just a modulesync entry

**Review by ekohl — COMMENTED** [review]
Given https://github.com/voxpupuli/puppet-tea/issues/19 and https://github.com/voxpupuli/puppet-tea/issues/52#issuecomment-567895531 I always considered this module as dead. Not sure we should really do a release.

**Comment by bastelfreak** [comment]
I'm going to close this PR because we're deprecating the module.

**Pull request closed without merging**